WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewController.swift

```diff
@@ -370,7 +370,8 @@ private extension ShippingLabelAddressFormViewController {
                                                                      state: .normal,
                                                                      keyboardType: .phonePad,
                                                                      textFieldAlignment: .leading) { [weak self] (newText) in
-            self?.viewModel.handleAddressValueChanges(row: row, newValue: newText)
+            let phone = newText?.filter { $0.isNumber }
```

Comment:
I'm getting picky here, but I checked the docs for `isNumber` and it seems to match many kinds of numeric characters, like `⅚` or `㊈`. This seems very unlikely to cause issues, so maybe it's ok to leave it like that, but I wanted to point that out.

Reply:
That's a valid point! I have updated the check to use a close range in 671efa984dfceeb3782992942bce42d19d3f6fea for safety.